enclave_manager/avalon_enclave_manager/enclave_manager.py

```diff
@@ -473,25 +504,57 @@ def start_enclave_manager(config):
                      "exiting Intel SGX Enclave manager: {err}")
         exit(1)
 
-    try:
-        sleep_interval = int(config["EnclaveManager"]["sleep_interval"])
-    except Exception as err:
-        logger.error("Failed to get sleep interval from config file. " +
-                     "Setting sleep interval to 10 seconds: %s", str(err))
-        sleep_interval = 10
+    if int(config["WorkloadExecution"]["sync_workload_execution"]) == 1:
+        # Binding with ZMQ Port
+        try:
+            socket = bind_zmq_socket(config.get("EnclaveManager")["zmq_port"])
+            logger.info("ZMQ Port hosted by Enclave")
+        except Exception as ex:
+            logger.exception("Failed to bind socket" +
+                             "shutting down enclave manager")
+            logger.error("Exception: {} args{} details{}".format(type(ex),
+                         ex.args, ex))
+            exit(1)
+        try:
+            while True:
+                # Wait for the next request
+                logger.info("Enclave Manager waiting for next request")
+                wo_id = socket.recv()
```

Comment:
@divyataori  We are using work order id in string format as primary key in all the tables. Since the work order id received from the message queue is a byte array, please convert to string and use it everywhere.
Hopefully after this change, CI job should pass without issues.